### PR TITLE
[New Rule] New BBR Rules - Part 3

### PR DESCRIPTION
**Pull request by w0rk3r**
## Issues

https://github.com/elastic/ia-trade-team/issues/158

## Summary

Adds 5 New BBR Rules:

* Kirbi File Creation
* Unusual Process Execution on WBEM Path
* Downloaded URL Files (Moved from Dev DR)
* WMI WBEMTEST Utility Execution
* WMIC Remote Command

Adjusted Mitre mappings on:

* Persistent Scripts in the Startup Directory

### CODE REVIEW

File: rules_building_block/credential_access_kirbi_file.toml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+from = "now-119m"
+interval = "60m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Kirbi File Creation"
+risk_score = 21
+rule_id = "b8f8da2d-a9dc-48c0-90e4-955c0aa1259a"
+severity = "low"
+tags = ["Domain: Endpoint", "OS: Windows", "Use Case: Threat Detection", "Tactic: Credential Access", "Rule Type: BBR"]
```
Comment (Aegrah) [inline rules_building_block/credential_access_kirbi_file.toml:25]:
```suggestion
tags = ["Domain: Endpoint", "OS: Windows", "Use Case: Threat Detection", "Tactic: Credential Access", "Data Source: Elastic Defend", "Rule Type: BBR"]
```

File: rules_building_block/execution_downloaded_url_file.toml [hunk 2]
```diff
@@ -1,36 +1,34 @@
...
 language = "eql"
 license = "Elastic License v2"
 name = "Downloaded URL Files"
-risk_score = 47
+risk_score = 21
 rule_id = "cd82e3d6-1346-4afd-8f22-38388bbf34cb"
-severity = "medium"
-tags = ["Domain: Endpoint", "OS: Windows", "Use Case: Threat Detection", "Tactic: Execution"]
+severity = "low"
+tags = ["Domain: Endpoint", "OS: Windows", "Use Case: Threat Detection", "Tactic: Execution", "Rule Type: BBR"]
```
Comment (Aegrah) [inline rules_building_block/execution_downloaded_url_file.toml:24]:
```suggestion
tags = ["Domain: Endpoint", "OS: Windows", "Use Case: Threat Detection", "Tactic: Execution", "Data Source: Elastic Defend", "Rule Type: BBR"]
```

File: rules_building_block/execution_wmi_wbemtest.toml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+from = "now-119m"
+interval = "60m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "WMI WBEMTEST Utility Execution"
+risk_score = 21
+rule_id = "d3551433-782f-4e22-bbea-c816af2d41c6"
+severity = "low"
+tags = ["Domain: Endpoint", "OS: Windows", "Use Case: Threat Detection", "Tactic: Execution", "Rule Type: BBR"]
```
Comment (Aegrah) [inline rules_building_block/execution_wmi_wbemtest.toml:24]:
```suggestion
tags = ["Domain: Endpoint", "OS: Windows", "Use Case: Threat Detection", "Tactic: Execution", "Data Source: Elastic Defend", "Rule Type: BBR"]
```

File: rules_building_block/lateral_movement_wmic_remote.toml [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+from = "now-119m"
+interval = "60m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "WMIC Remote Command"
+risk_score = 21
+rule_id = "f59668de-caa0-4b84-94c1-3a1549e1e798"
+severity = "low"
+tags = ["Domain: Endpoint", "OS: Windows", "Use Case: Threat Detection", "Tactic: Lateral Movement", "Rule Type: BBR"]
```
Comment (Aegrah) [inline rules_building_block/lateral_movement_wmic_remote.toml:24]:
```suggestion
tags = ["Domain: Endpoint", "OS: Windows", "Use Case: Threat Detection", "Tactic: Lateral Movement", "Data Source: Elastic Defend", "Rule Type: BBR"]
```

**Review by Aegrah — APPROVED** [review]
LGTM, just added a few tags.

**Review by Samirbous — APPROVED** [review]

**Review by brokensound77 — APPROVED** [review]

**Pull request merged**